src/sagemaker/remote_function/client.py

```diff
@@ -57,6 +59,7 @@
 logger = logging_config.get_logger()
 
 
+@_telemetry_emitter(feature=Feature.REMOTE_FUNCTION, func_name="remote_function.remote")
```

Comment:
The full path seems to be sagemaker.remote_function.client.remote, given the import. Wondering if we should use the full function name here, e.g. `remote_function.client.remote`.

If it does not matter for the telemetry, then no need to update. 

```
from sagemaker.remote_function.client import (
    remote,
...
)
```